java/pom.xml

```diff
@@ -679,6 +675,14 @@
         <module>gandiva</module>
       </modules>
     </profile>
+
+    <profile>
```

Comment:
Maybe I'm missing something but I thought that the shaded artifacts are always produced alongside the normal ones? So I'm not sure why we need a profile

Reply:
My intent was to allow testing the the shaded artifact with downstream modules.  (see the changes to the ci shell scripts where I tried to excercise this profile).

Reply:
I get it now, seems fine to me